internal/communicator/ssh/communicator.go

```diff
@@ -877,14 +877,17 @@ func (c *bastionConn) Close() error {
 	return c.Bastion.Close()
 }
 
-func quoteShell(args []string, targetPlatform string) (string, error) {
+func quoteShell(args []string, targetPlatform string, scriptPath string) (string, error) {
 	if targetPlatform == TargetPlatformUnix {
 		return shquot.POSIXShell(args), nil
 	}
 	if targetPlatform == TargetPlatformWindows {
-		return shquot.WindowsArgv(args), nil
+		cmd := shquot.WindowsArgv(args)
+		if (strings.HasPrefix(cmd, `"`) && strings.HasSuffix(scriptPath, ".ps1")) {
```

Comment:
If `cmd` is prefixed with `"` and the script is PowerShell then we need to prefix the command with `&` so the shell will execute the command rather than treat it as a string.